src/axom/inlet/Field.cpp

```diff
@@ -245,12 +245,14 @@ std::shared_ptr<Field> Field::range(double startVal, double endVal)
   return shared_from_this();
 }
 
-bool Field::get(bool& value)
+template <>
+bool Field::get<bool>()
 {
   axom::sidre::View* valueView = m_sidreGroup->getView("value");
   if(valueView == nullptr)
   {
-    return false;
+    SLIC_ERROR("[Inlet] Field does not contain a value");
+    setWarningFlag(m_sidreRootGroup);
```

Comment:
This warning flag is only checked during `Inlet::verify()` so that all the verifications are run before finally failing.  I don't think it makes sense to mark this.  Just assume the users is going to error out at this point.

Reply:
We should probably rename this to be more specific, like `setSchemaWarningFlag` or `setSchemaStepThrewWarningFlag`.